components/Common/Pagination/index.tsx

```diff
@@ -0,0 +1,82 @@
+import { ArrowRightIcon, ArrowLeftIcon } from '@heroicons/react/20/solid';
+import type { FC } from 'react';
+import { FormattedMessage, useIntl } from 'react-intl';
+
+import Button from '@/components/Common/Button';
+
+import styles from './index.module.css';
+import { useGetPageElements } from './useGetPageElements';
```

Comment:
In general style imports should be the last thing imported. What happens if you change this to this:

```tsx
import { useGetPageElements } from './useGetPageElements';

import styles from './index.module.css';
```

Reply:
Awesome, thank you. I had to modify the path of the `useGetPageElements` import for this to work, as `ESLint` was the one formatting that import statement and placing it below the `styles` one.